### DISCUSSION TITLE
How to turn off images in the article list？

### DISCUSSION THREAD

**Discussion by zhaozhiliao** [thuliteio/doks#850] [Q&A] [answered]
1.I have updated the latest version, but I like the previous blog list style, how to turn off the thumbnail display and only show the list without image style.
2. Another question, is it possible to create two blog type folders
![1](https://qnimg.zhaobanxian.top/img/Snipaste_2022-07-22_22-52-26.png)
3.Why some blog posts are centered and some are not. For example.
[Normal](https://zhaobanxian.top/blog/%E5%A6%82%E4%BD%95%E5%86%99%E4%B8%80%E7%AF%87%E5%8F%8A%E6%A0%BC%E7%9A%84%E9%9C%80%E6%B1%82%E6%96%87%E6%A1%A3/)
[Centered Articles](https://zhaobanxian.top/blog/02_%E5%B8%B8%E7%94%A8%E8%BD%AF%E4%BB%B6%E6%8E%A8%E8%8D%90/)

**Comment by zhaozhiliao** [comment]
i need help~~~~~Browse ten times a day

**Comment by h-enk** [comment, marked as answer]
> 1. I have updated the latest version, but I like the previous blog list style, how to turn off the thumbnail display and only show the list without image style.

Not supported yet — will be part of the upcoming release (will make it an option to choose between the two)

> 2. Another question, is it possible to create two blog type folders

Just copy the `./content/en/blog/` folder + paste it as e.g. `./content/en/news/`. Note, will also require updating `./assets/scss/common/_global.scss` — add: 

```
.news .content {
  padding-top: 1rem;
  padding-bottom: 3rem;
}
```
I will make this easier in the next release ( so you could use something like `npm run create -- --kind blog news`).

> 3. Why some blog posts are centered and some are not. For example.

Not sure why `<div class="text-center"></div>` is added — did you do that manually?

![Snag_ab5b6f7](https://user-images.githubusercontent.com/3902872/184389165-744b9658-2496-4484-9671-eeee0a285127.png)

**Reply by zhaozhiliao** [reply]
Thank you very much, can I translate your tutorial document into Chinese, I would like to introduce it to other friends. such as:[tutorial](https://zhaobanxian.top/dokscn/00%E6%A6%82%E8%BF%B0/00%E5%AE%89%E8%A3%85%E8%A6%81%E6%B1%82/)
